clio-server/src/test/scala/org/broadinstitute/clio/server/dataaccess/PersistenceDAOSpec.scala

```diff
@@ -1,20 +1,22 @@
 package org.broadinstitute.clio.server.dataaccess
 
-import org.broadinstitute.clio.server.dataaccess.elasticsearch.{
-  DocumentMock,
-  ElasticsearchIndex
-}
+import org.broadinstitute.clio.server.dataaccess.elasticsearch.{DocumentMock, ElasticsearchIndex}
 import org.broadinstitute.clio.server.dataaccess.elasticsearch.Elastic4sAutoDerivation._
-
 import com.sksamuel.elastic4s.circe._
 import io.circe.parser._
 import org.scalatest.{AsyncFlatSpec, Matchers}
-
 import java.nio.file.Files
+import java.util.UUID
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
 
 class PersistenceDAOSpec extends AsyncFlatSpec with Matchers {
```

Comment:
The way we do this elsewhere is to change `AsyncFlatSpec` to `TestKit(ActorSystem("some-name")) with AsyncFlatSpecLike with BeforeAndAfterAll`. The test kit class will handle making the system implicit, and also give you a `shutdown()` method to call in `afterAll()` to make sure everything cleans up correctly. Example [here](https://github.com/broadinstitute/clio/blob/develop/clio-client/src/test/scala/org/broadinstitute/clio/client/BaseClientSpec.scala).

Reply:
OK, thanks. I still had to construct a private implicit materializer though.